README.md

```diff
@@ -0,0 +1,68 @@
+# Awesome-books_JavaScript-objects
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+> In this project, we refactor the awesome books project to use ES6 syntax and package management using npm.
+
+## Screenshot
+
+![app-screenshot](./screenshot.png)
+
+## Built With
+
+- HTML/CSS, Javascript
+- Git/Github, CLI, npm
+- Luxon.js
+
+## Live Demo
+
+[Live Demo Link](https://jssol.github.io/awesome-books-es6/)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+To get the project up and running you need a browser.
+[The project live link](#Live-Demo-Link)
+
+### Install
+
+To get the project locally, run `git clone git@github.com:jssol/awesome-books-es6.git`
+
+### Usage
+
+This project helps you store your favorite books right in your navigator.
+
+### Run tests
+
+For:
+
+- Eslint: run `npx eslint .`
+- Eslint: run `npx hint .`
```

Comment:
- [ ] I think the correct linter being run here is Webhint. I'd recommend you fix the typo error.